bot.js

```diff
@@ -65,110 +65,68 @@ if (!process.env.clientId || !process.env.clientSecret || !process.env.PORT) {
+            glitch_domain: process.env.PROJECT_DOMAIN,
+            layout: 'layouts/default'
+        });
     });
-  })
-  // Set up a simple storage backend for keeping a record of customers
-  // who sign up for the app via the oauth
-  require(__dirname + '/components/user_registration.js')(controller);
-
-  // Send an onboarding message when a new team joins
-  require(__dirname + '/components/onboarding.js')(controller);
-
-  var normalizedPath = require("path").join(__dirname, "skills");
-  require("fs").readdirSync(normalizedPath).forEach(function(file) {
-    require("./skills/" + file)(controller);
-  });
-
-  // This captures and evaluates any message sent to the bot as a DM
-  // or sent to the bot in the form "@bot message" and passes it to
-  // Botkit Studio to evaluate for trigger words and patterns.
-  // If a trigger is matched, the conversation will automatically fire!
-  // You can tie into the execution of the script using the functions
-  // controller.studio.before, controller.studio.after and controller.studio.validate
-  if (process.env.studio_token) {
-      controller.on('direct_message,direct_mention,mention', function(bot, message) {
-          controller.studio.runTrigger(bot, message.text, message.user, message.channel, message).then(function(convo) {
-              if (!convo) {
-                  // no trigger was matched
-                  // If you want your bot to respond to every message,
-                  // define a 'fallback' script in Botkit Studio
-                  // and uncomment the line below.
-                  // controller.studio.run(bot, 'fallback', message.user, message.channel);
-              } else {
-                  // set variables here that are needed for EVERY script
-                  // use controller.studio.before('script') to set variables specific to a script
-                  convo.setVar('current_time', new Date());
-              }
-          }).catch(function(err) {
-              bot.reply(message, 'I experienced an error with a request to Botkit Studio: ' + err);
-              debug('Botkit Studio: ', err);
-          });
-      });
-  } else {
-      console.log('~~~~~~~~~~');
-      console.log('NOTE: Botkit Studio functionality has not been enabled');
-      console.log('To enable, pass in a studio_token parameter with a token from https://studio.botkit.ai/');
-  }
-}
 
+    // Set up a simple storage backend for keeping a record of customers
+    // who sign up for the app via the oauth
+    require(__dirname + '/components/user_registration.js')(controller);
+
+    // Send an onboarding message when a new team joins
+    require(__dirname + '/components/onboarding.js')(controller);
+
+    const normalizedPath = require("path").join(__dirname, "skills");
 
+    require("fs").readdirSync(normalizedPath).forEach((file) => {
+        require("./skills/" + file)(controller);
+    });
 
+} else {
 
+    webserver.get('/', (req, res) => {
```

Comment:
If we don't have a `clientId` or `clientSecret`, just show a very basic message.